### ISSUE TITLE
Tab order on equipement edit

### ISSUE THREAD

**Issue by schtibal** [submersion-app/submersion#444]
When editing or adding new equipment on the desktop version, tab order goes between the equipment edit tab (on the right) and the equipment list tab (on the left). Pressing tab should be constrained to only the active window panel.
